app/scripts/modules/amazon/src/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -349,9 +349,10 @@ angular
                 launchTemplateData.instanceMarketOptions.spotOptions &&
                 launchTemplateData.instanceMarketOptions.spotOptions.maxPrice;
               const { ipv6AddressCount } =
-                launchTemplateData.networkInterfaces &&
-                launchTemplateData.networkInterfaces.length &&
-                launchTemplateData.networkInterfaces[0];
+                (launchTemplateData.networkInterfaces &&
+                  launchTemplateData.networkInterfaces.length &&
+                  launchTemplateData.networkInterfaces[0]) ||
+                0;
```

Comment:
This change is not needed. It is ok if this ends up being `undefined` since we are just checking if it exists below.

Reply:
This change was added in order of the new tests in `awsServerGroupCommandBuilder` -> `buildServerGroupCommandFromExisting` to pass. 

It might be better to modify the test data and remove this code change. Will do that.